internal/deviations/README.md

```diff
@@ -0,0 +1,88 @@
+## Guidelines to add deviations to FNT tests
+
+* Add the deviation to the `Deviation` message in the [proto/metadata.proto](https://github.com/openconfig/featureprofiles/blob/main/proto/metadata.proto) file.
+
+```
+message Deviations {
+...
+// Device does not support fragmentation bit for traceroute.
+bool traceroute_fragmentation = 2;
+...
+}
+```
+
+* Run `make proto/metadata_go_proto/metadata.pb.go` from your featureprofiles root directory to generate the Go code for the added proto fields.
+
+```
+$ make proto/metadata_go_proto/metadata.pb.go
+mkdir -p proto/metadata_go_proto
+# Set directory to hold symlink
+mkdir -p protobuf-import
+# Remove any existing symlinks & empty directories
+find protobuf-import -type l -delete
+find protobuf-import -type d -empty -delete
+# Download the required dependencies
+go mod download
+# Get ondatra modules we use and create required directory structure
+go list -f 'protobuf-import/{{ .Path }}' -m github.com/openconfig/ondatra | xargs -L1 dirname | sort | uniq | xargs mkdir -p
+go list -f '{{ .Dir }} protobuf-import/{{ .Path }}' -m github.com/openconfig/ondatra | xargs -L1 -- ln -s
+protoc -I='protobuf-import' --proto_path=proto --go_out=./ --go_opt=Mmetadata.proto=proto/metadata_go_proto metadata.proto
+goimports -w proto/metadata_go_proto/metadata.pb.go
+```
+
+* Add the accessor function for this deviation to the [internal/deviations/deviations.go](https://github.com/openconfig/featureprofiles/blob/main/internal/deviations/deviations.go) file. This function will need to accept a parameter `dut` of type `*ondatra.DUTDevice` to lookup the deviation value for a specific dut. This accessor function must call `lookupDUTDeviations` and return the deviation value. Test code will use this function to access deviations.
+	* If default value of deviation is same as default value for proto field, accessor method can directly call the `Get*()` method for deviation field. For e.g. for `bool` fields, the default is `false`, and the accessor method will have one line `return lookupDUTDeviations(dut).GetTracerouteFragmentation()`.
```

Comment:
Add some missing "the" words:
If *the* default value of the deviation is the same as *the* default value for *the* proto field, *the* accessor method...

Reply:
done